lib/msf/core/exploit/remote/http_client.rb

```diff
@@ -373,22 +372,34 @@ def send_request_raw(opts = {}, timeout = 20, disconnect = false)
   # Connects to the server, creates a request, sends the request,
   # reads the response
   #
+  # If a +Msf::Exploit::Remote::HTTP::HttpCookieJar+ instance is passed in the +opts+ dict under a 'cookie' key, said CookieJar will be used in
+  # the request instead of the module +cookie_jar+
+  #
   # Passes `opts` through directly to {Rex::Proto::Http::Client#request_cgi}.
   # Set `opts['keep_cookies']` to keep cookies from responses for reuse in requests.
+  # Cookies returned by the server will be stored in +cookie_jar+
+  #
+  # +expire_cookies+ will control if +cleanup+ is called on any passed +Msf::Exploit::Remote::HTTP::HttpCookieJar+ or the client cookiejar
```

Comment:
I think this documentation needs updated now that `expire_cookies` isn't a a parameter

Reply:
Let's leave this to a follow up PR